src/clients/api/queries/getMintableVai/types.ts

```diff
@@ -0,0 +1,12 @@
+import BigNumber from 'bignumber.js';
+
+import { VaiUnitroller } from 'types/contracts';
+
+export interface GetMintableVaiInput {
+  vaiControllerContract: VaiUnitroller;
+  accountAddress: string;
+}
```

Comment:
```suggestion
export interface GetMintableVaiInput {
  accountAddress: string;
}
```

Reply:
The function needs the contract to be passed to it